pnpm-lock.yaml

```diff
@@ -1,4 +1,4 @@
-lockfileVersion: '6.1'
+lockfileVersion: '6.0'
```

Comment:
I wasn't sure why my pnpm (v8.6.2) downgraded this, but it seems this is [intentional](https://github.com/pnpm/pnpm/releases/tag/v8.6.2) behavior.